test/integration/shared/controls/client.rb

```diff
@@ -0,0 +1,49 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "yaml"
+
+forseti_server_vm_internal_dns = attribute("forseti-server-vm-internal-dns")
+forseti_version = "2.24.0"
```

Comment:
Should this be 2.25?

Reply:
Nice catch! Somehow I managed to not update the client/server controls that were being used by the install simple fixture.. so there were multiple controls for client & server. I'm guessing that the tests got lucky and used the version of the controls that had 2.25.0 as the required version. I've made the changes locally and re-ran the tests, which are passing. Incoming commit :)

Reply:
The changes aren't easy to see in the PR, I guess GitHub isn't a fan of symlinks. You can see the controls used by the install_simple [here](https://github.com/forseti-security/terraform-google-forseti/tree/feature/shared-controls/test/integration/install_simple/controls).